### PR TITLE
Feature Info Polish

### PR DESCRIPTION
**Pull request by yvonnesjy**
- Widened the modal when screen fits
- Adjusted layout and spacing within feature info
- Added animation to content when opened
- Fixed iFrame height calculation

Demo
- LEO
image
image

- Communities
image
image

- Table
image
image

### CODE REVIEW

File: src/js/views/maps/FeatureInfoView.js [hunk 1]
```diff
@@ -309,24 +310,15 @@ define(
...
-          if (!iFrame) return;
-          if ((!height && height !== 0) || height < 0) {
-            height = iFrame.contentWindow.document.body.scrollHeight + 5;
-          }
-          if (limit) {
-            const maxHeight = window.innerHeight - 275;
-            height = height > maxHeight ? maxHeight : height;
-          }
-          iFrame.style.height = height + "px";
+          const maxHeight = window.innerHeight - 320;
```
Comment (ianguerin) [inline src/js/views/maps/FeatureInfoView.js:317]: For "[magic numbers](https://en.wikipedia.org/wiki/Magic_number_%28programming%29)" like 320, can we have a comment associated describing what/why it is this value

File: src/js/views/maps/FeatureInfoView.js [hunk 2]
```diff
@@ -309,24 +310,15 @@ define(
...
+          let height = iFrame.contentWindow.document.getElementById("content").scrollHeight + 24;
+          height = height > maxHeight ? maxHeight : height;
```
Comment (ianguerin) [inline src/js/views/maps/FeatureInfoView.js:319]: Alternatively:
```
const height = Math.min(maxHeight, iFrame.contentWindow.document.getElementById("content").scrollHeight + 24); 
```

**Review by ianguerin — APPROVED** [review]

File: src/js/views/maps/FeatureInfoView.js (hunk 2 above)
Comment (yvonnesjy) [reply src/js/views/maps/FeatureInfoView.js]: Done

**Comment by yvonnesjy** [comment]
> button text overflows out of the button on small screens

@robyngit Interesting. Can you check if your version has `text-wrap: pretty;` under `.feature-info__zoom-button, .feature-info__layer-details-button`? I added it for the same overflow issue🤔

**Comment by robyngit** [comment]
Another FF vs Chrome issue! https://caniuse.com/mdn-css_properties_text-wrap_pretty

**Comment by yvonnesjy** [comment]
DOH! Fixed!

**Pull request merged**